device_backends/xdma/src/XdmaBackend.cc

```diff
@@ -39,15 +39,8 @@ namespace ChimeraTK {
       }
     }
 
-    // (Re-)Open the event files which are needed
+    // (Re-)Open the event files only when you acquire an accessaor.
     std::for_each(_eventFiles.begin(), _eventFiles.end(), [](auto& eventFile) { eventFile = nullptr; });
```

Comment:
These the comment is not quite correct. You open the event file when startInterruptHandlingThread is called (the Thread lives inside the EventFile class).
You can remove both the comment and the code line. Cleaning in open() is not useful. It is happening close() and probably should happen in setException (to be discussed), but open is definitely the wrong place.
```suggestion
```

Reply:
done